lang/java/avro/src/test/java/org/apache/avro/TestSchemaBuilder.java

```diff
@@ -561,6 +561,17 @@ public void testEnum() {
     Assert.assertEquals(expected, schema);
   }
 
+  @Test
+  public void testEnumWithDefault() {
+    List<String> symbols = Arrays.asList("a", "b");
+    String enumDefault = "a";
+    Schema expected = Schema.createEnum("myenum", null, null, symbols, enumDefault);
+    expected.addProp("p", "v");
+    Schema schema = SchemaBuilder.enumeration("myenum")
+      .prop("p", "v").defaultSymbol(enumDefault).symbols("a", "b");
```

Comment:
So this particular code decision means that the default must be set before calling .symbols(). It will not work the other way around.